lib/vanagon/engine/pooler.rb

```diff
@@ -35,12 +48,12 @@ def select_target
         response = Vanagon::Utilities.http_request(
           "#{@pooler}/vm",
           'POST',
-          '{"' + @platform.vmpooler_template + '":"1"}',
+          '{"' + @build_host_type + '":"1"}',
```

Comment:
Random question -- why not use `build_host_type` here instead of `@build_host_type`?